.env.example

```diff
@@ -2,6 +2,5 @@ DB_USERNAME=postgres
 DB_PASSWORD=db_password
 DB_NAME=barefoot_nomad
 DB_HOST=127.0.0.1
-PORT=
```

Comment:
Why did you remove the `PORT`

Reply:
Well now i understand the use of `env.example`.